### ISSUE TITLE
508-defect-4 [COGNITION]: App Directory - Consider reviewing technical language for OneRecord app additional information

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#16465]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made. This is a WCAG 2 AAA improvement item that will benefit users, but can be closed if the description is something we cannot change.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user who might have a cognitive issue or be unfamiliar with technical acronyms like FHIR, I would appreciate the technical language in the OneRecord app be reviewed to see if it there are opportunities for plain language.

## Details

This issue is a WCAG 2 Level AAA enhancement. The OneRecord has a lot of technical terms and acronyms that would benefit from a content review for plain language.

## Acceptance Criteria

- [ ] Content is reviewed for plain language, changed if a review warrants changes

## WCAG or Vendor Guidance (optional)

* [Unusual Words: Understanding SC 3.1.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/meaning-idioms.html)
* [H54: Using the dfn element to identify the defining instance of a word](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/H54)

## Screenshots or Trace Logs

<img width="707" alt="Screen Shot 2020-11-19 at 8 20 05 PM" src="https://user-images.githubusercontent.com/934879/99750639-5b527380-2aa6-11eb-8ded-09b074f08735.png">

**Comment by ricetj** [comment]
@1Copenut I agree that the text here is not as user friendly as one would hope, but this is coming straight from the 3rd party app (in this example one health), so we really don't want to edit or rework any content they are marketing to users.

**Comment by 1Copenut** [comment]
@ricetj I wondered if that was the case. Given that fact, and the SC is a WCAG 2 AAA (beyond basic Section 508 compliance). Go ahead and close this issue when you're ready.
